### DISCUSSION TITLE
Transition groups

### DISCUSSION THREAD

**Discussion by samuk** [MurmurationsNetwork/MurmurationsLibrary#93] [General]
I'd be up for doing a schema & endpoints for https://transitiongroups.org/api/ if someone is up for helping

**Comment by geoffturk** [comment]
Thanks for reaching out. Yes, we would be glad to help you set up a schema and get your endpoints in the Murmurations index.

How would you prefer to kick things off? We can talk it through on our [Telegram channel](https://t.me/joinchat/JvotB0kuxrjFgvYszbNvZw), or perhaps schedule a call if you prefer. Let us know either way.

ping: @olisb

**Comment by olisb** [comment]
Hey @samuk - very excited to hear you're interested in doing this, just let us know how / when where you'd prefer to chat? 
Are you in the UK? I'm in London and Geoff is in France...

**Comment by samuk** [comment]
A call at some point in the new year would be great.

**Comment by olisb** [comment]
Cool - shall we say 3pm uk time on Thursday 5th Jan? We can use my BBB room...

**Comment by samuk** [comment]
Yes I think that works.

Are you going to this tomorrow? https://www.catalist.network/event/mapathon22

**Comment by olisb** [comment]
yes, I'll be joining at 5pm GMT

**Comment by olisb** [comment]
and possibly 9pm GMT too

**Comment by geoffturk** [comment]
@samuk We put together this new page in our docs after today's session at Mapathon22:

https://docs.murmurations.network/about/ontology-composability.html

It seems like you already get the concept of what we're doing pretty well, but perhaps this might add a bit more.

Feel free to ping us before we speak in Jan if you have any questions about setting up a schema and importing data for Transition Network. 

🤘

**Comment by olisb** [comment]
Hey @samuk let me know the best way to contact you privately to share the meeting link for the 5th Jan at 3pm GMT? Thanks
